plugins/remotelog/plugin.go

```diff
@@ -105,7 +105,7 @@ func run(plugin *node.Plugin) {
 
 func sendLogMsg(level logger.Level, name string, msg string) {
 	m := logMessage{
-		banner.AppVersion,
+		banner.SimplifiedAppVersion,
```

Comment:
I think here we should actually send the `AppVersion`. Then we can specifically see which version is used.